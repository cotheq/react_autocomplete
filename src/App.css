.App {
    display: flex;
    width: 100%;
    height: 100%;
    justify-content: center;
    align-items: center;
}

.form_container {
    width: 343px;
    height: 278px;
}