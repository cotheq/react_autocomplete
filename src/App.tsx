import React from 'react';
import logo from './logo.svg';
import Form from "./components/Form/Form"
import './App.scss';

function App() {
  return (
    <div className="App">
        <div className="form_container">
            <Form />
        </div>
    </div>
  );
}

export default App;
