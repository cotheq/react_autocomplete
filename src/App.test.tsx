// describe('Testing sum', () => {
//     function sum(a: number, b: number) {
//        return a + b;
//     }

//     it('should equal 4',()=>{
//        expect(sum(2,2)).toBe(4);
//       })

// });


import React from 'react';
import { render } from "@testing-library/react"
import App from "./App"
import ReactDOM from 'react-dom';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
  });

it('renders correctly', () => {
  const {asFragment} = render(<App />)
  expect(asFragment()).toMatchSnapshot();
});