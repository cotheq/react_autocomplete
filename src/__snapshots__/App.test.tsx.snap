// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="form_container"
    >
      <div
        class="search_form"
        data-testid="form_container"
      >
        <input
          data-testid="input"
          placeholder="Search"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
